README.md

```diff
@@ -66,14 +66,15 @@ boradcasted on Redis pub/sub.
 
 All messages has signer id field which is specify by signer public key.
 
-| Name            | payload | Description                                                  |
-| --------------- | ------- | ------------------------------------------------------------ |
-| nodevss         | NodeVss | Each signers send VSS in Key Generation Protocol.            |
-| candidateblock  | Block   | Round master broadcasts to signer network a candidate block. |
-| blockvss        | BlockVSS| Send vss for random secret.                                  |
-| blocksig        | LocalSig| Broadcast local sig.                                         |
-| completedblock  | Block   | Round master broadcasts completed block.                     |
-| roundfailure    |         | Round master notifies the round is failure.                  |
+| Name              | payload        | Description                                                  |
+| ----------------- | -------------- | ------------------------------------------------------------ |
+| nodevss           | NodeVss        | Each signers send VSS in Key Generation Protocol.            |
+| candidateblock    | Block          | Round master broadcasts to signer network a candidate block. |
+| blockvss          | BlockVSS       | Send vss for random secret.                                  |
+| blockparticipants | Vec<PublicKey> | Round master notify signature issuing protocol is going to be executed with the signers who are represented in payload keys |
+| blocksig          | LocalSig       | Broadcast local sig.                                         |
+| completedblock    | Block          | Round master broadcasts completed block.                     |
+| roundfailure      |                | Round master notifies the round is failure.                  |
 
 
 ## Round
```

Comment:
a sequence diagram is useful to understand following flow.

Reply:
I' ll try to create the diagram.

Reply:
I added diagram.